migrations/1_initial_migration.js

```diff
@@ -1,5 +1,5 @@
 const Migrations = artifacts.require("./Migrations.sol")
 
 module.exports = function(deployer) {
-  deployer.deploy(Migrations)
+  deployer.deploy(Migrations, { gas: 300000 })
```

Comment:
why is this needed, any clue?

Reply:
The reason why I included this is that I hope we can get away with cheaper gas price for deployment. The default gas limit is currently very high, about 6000000, so a miner might ignore our transaction until there is a block that has 6M gas available at a price that we give, in which case we need a higher gas price to deploy at the same speed as with a lower gas limit.
I have to say that I'm not sure it really works like this, so maybe this is overthinking the problem. Considering we are not going to deploy this contract again, we could just remove this hardcoded gas limit.